src/XrdPosix/XrdPosixMap.hh

```diff
@@ -35,6 +35,7 @@
 #include "XrdCl/XrdClXRootDResponses.hh"
 
 #include <cstdint>
+#include <sys/stat.h>
```

Comment:
Again, please forward declare struct stat and move the include to the cc file where the object is used.

Reply:
Done.